common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftTokenErrorResponse.java

```diff
@@ -41,6 +41,9 @@
     @SerializedName("correlation_id")
     private String mCorrelationId;
 
+    @SerializedName("oAuth_metadata")
+    private String mOAuthErrorMetadata;
```

Comment:
Curious. Any example of error metadata?

Reply:
I think currently we don't get any, but eventually we would I guess, just moved this from `BrokerErrorResponse`